WTWSMS/events/WtWSMS_holy_orders_events.txt

```diff
@@ -86,18 +86,19 @@ narrative_event = {
 			create_character = {
 				random_traits = yes
 				dynasty = random
-				religion = semitic_pagan_reformed
+				religion = semitic_pagan
```

Comment:
Under your code, Reformed Semitic Pagans spawn no matter what, even when they don't exist in-game. That doesn't make any sense.